torch/quantization/fake_quantize.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from torch.nn import Module
+from .._jit_internal import weak_module
+from .observer import Observer, observer
+
+@weak_module
+class FakeQuantize(Module):
+    ''' Simulate the quantize and dequantize operations in training time.
+    Args:
+        `qconfig`: object that encodes configuration info for quantization
+        `observer_module`: Observer module that records stats of weights and
+        activations
+        `calcqparam`: A function that calculates quantization parameters
+        given the stats
+    '''
+
+    def __init__(self, qconfig, quant_min=-128, quant_max=127, quant_delay=0):
+        super(FakeQuantize, self).__init__()
+        self.observer = qconfig.activation()
+        self.qconfig = qconfig
+        self.quant_min = quant_min
+        self.quant_max = quant_max
```

Comment:
We need to ensure that quant_min and quant_max are consistent with dtype: We should not have torch.qint8 as the dtype and have qmin=0 and qmax=255 for example.

Reply:
still no addressed. They should start with None as default values and you can pull it out of DTYPE_RANGE